stl/inc/list

```diff
@@ -185,33 +185,33 @@ public:
         return *this;
     }
 
-    _List_const_iterator operator--(int) {
+    _List_const_iterator operator--(int) noexcept {
         _List_const_iterator _Tmp = *this;
         --*this;
         return _Tmp;
     }
 
-    _NODISCARD bool operator==(const _List_const_iterator& _Right) const {
+    _NODISCARD bool operator==(const _List_const_iterator& _Right) const noexcept {
 #if _ITERATOR_DEBUG_LEVEL == 2
         _STL_VERIFY(this->_Getcont() == _Right._Getcont(), "list iterators incompatible");
 #endif // _ITERATOR_DEBUG_LEVEL == 2
 
         return this->_Ptr == _Right._Ptr;
     }
 
-    _NODISCARD bool operator!=(const _List_const_iterator& _Right) const {
+    _NODISCARD bool operator!=(const _List_const_iterator& _Right) const noexcept {
         return !(*this == _Right);
     }
 
 #if _ITERATOR_DEBUG_LEVEL == 2
-    friend void _Verify_range(const _List_const_iterator& _First, const _List_const_iterator& _Last) {
+    friend void _Verify_range(const _List_const_iterator& _First, const _List_const_iterator& _Last) noexcept {
         _STL_VERIFY(_First._Getcont() == _Last._Getcont(), "list iterators in range are from different containers");
     }
 #endif // _ITERATOR_DEBUG_LEVEL == 2
 
     using _Prevent_inheriting_unwrap = _List_const_iterator;
 
-    _NODISCARD _List_unchecked_const_iterator<_Mylist> _Unwrapped() const {
+    _NODISCARD _List_unchecked_const_iterator<_Mylist> _Unwrapped() const noexcept {
         return _List_unchecked_const_iterator<_Mylist>(this->_Ptr, static_cast<const _Mylist*>(this->_Getcont()));
     }
 

```

Comment:
`_Seek_to` immediately below isn't being marked `noexcept`, but for the other container iterators it is. (Thanks @BillyONeal for teaching me how to use the GitHub Issues & Pull Requests extension for VSCode which displays full context.)